IAMOTZ/ievents
# #152987746 Set up eslint

### What does this PR do?
Set up Eslint for the project using Airbnb style guide.
#### Description of Task to be completed?
Set up Esllint for server side development
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- create a test Javascript file
- run `npm run lint path/to/test-file` to lint the file
- the conventions that contradict Airbnb style guide would appear in the command line
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152987746
#### Screenshots (if appropriate)
None
#### Questions:
None
